internal/gateway/http_client.go

```diff
@@ -0,0 +1,103 @@
+package gateway
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+)
+
+type HttpClient struct {
+	Client   *http.Client
+	Endpoint string
+	Headers  map[string]string
+	Params   map[string]interface{}
+	Body     io.Reader
```

Comment:
メソッドチェーンでやるならここら辺プライベートでも良いかなと思ったけど、まあ時間ないならどちらでも

```suggestion
	Client   *http.Client
	endpoint string
	headers  map[string]string
	params   map[string]interface{}
	body     io.Reader
```

Reply:
そうやな、すまん修正しておく！

Reply:
https://github.com/cocoide/commitify/pull/67/commits/2ddf00a668c63d953e08e5ad3a2fde9eb3af673a
done here